docs/contributing/tasks.md

```diff
@@ -34,7 +34,7 @@ npm scripts are defined in `package.json`. These trigger a number of Gulp tasks.
 **`npm run build:app` will do the following:**
 
 - clean the `./public` folder
-- output files into `./public`, or another location via the `--destination` flag
```

Comment:
I don't do this personally, but is it worth checking whether anybody else makes use of `--destination` to, say, build to a temp directory while developing?

Reply:
Looking good for removal so far

Split out from this PR now, but I'm happy with `destPath` being configurable per destination:

* https://github.com/alphagov/govuk-frontend/pull/3403

Reply:
I think if nobody's doing anything other than standard builds anyway, let's not bother. We can always pop it back in quick enough if folks complain.